pkg/monitor/cluster/clustervmpower.go

```diff
@@ -0,0 +1,38 @@
+package cluster
+
+import (
+	"context"
+
+	"github.com/Azure/ARO-RP/pkg/util/stringutils"
+	"github.com/Azure/ARO-RP/pkg/util/virtualmachines"
+)
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+func (mon *Monitor) emitVMPowerStatus(ctx context.Context) (stopped bool, err error) {
```

Comment:
Personally I am not a fan of the named return values as 
- it a bit harder to ready what value you are actually returning
- it makes it easier to shadow values (like err)

Reply:
What is the use of this metric now considering my previous comment?

Reply:
@mjudeikis I suppose it would be of very little use :slightly_smiling_face: